pommerman/cli/run_battle.py

```diff
@@ -46,7 +48,7 @@ def run(args, num_times=1, seed=None):
     if record_json_dir and not os.path.isdir(record_json_dir):
         os.makedirs(record_json_dir)
 
-    def _run(seed, record_pngs_dir=None, record_json_dir=None):
```

Comment:
why did you remove the record_json_dir argument?

Reply:
It is defined in the file's global scope and b/c of the way it works now there is no reason to create subdirectories for each iteration. The merge JSON function will create a time-stamped file like `game_state_20180528091945.json`. It will then remove all of the old single step files.